src/realm.json

```diff
@@ -1633,6 +1633,25 @@
                     }
                 }
             ]
+        },
+        {
+            "id": "fa4b3a43-7a51-449c-9937-c992f582dd1d",
+            "name": "aws-groups",
+            "description": "SAML custom scope for formatting user groups for AWS",
+            "protocol": "saml",
+            "attributes": {
+                "include.in.token.scope": "true",
+                "display.on.consent.screen": "true"
+            },
+            "protocolMappers": [
+                {
+                    "id": "a87cb1d0-214b-4e92-8b93-562efb69f36d",
+                    "name": "aws-groups",
+                    "protocol": "saml",
+                    "protocolMapper": "aws-saml-group-mapper",
+                    "consentRequired": false
+                }
+            ]
```

Comment:
If we do want to pull this into the baseline (open question I think) I would lean towards not adding this into the realm given that we don't expect this default one to be used by known asks for it.